### PR TITLE
feat(offer): add companyAdminRoles to service & app

### PR DESCRIPTION
**Pull request by Phil91**
Refs: CPLP-2521
corresponding Backend-PR: [369](https://github.com/catenax-ng/tx-portal-backend/pull/369)

### CODE REVIEW

**Review by evegufy — APPROVED** [review]

File: charts/portal/values.yaml [hunk 1]
```diff
@@ -407,6 +407,8 @@ backend:
       status1: "ACTIVE"
     swaggerEnabled: false
     technicalUserProfileClient: "technical_roles_management"
+    companyAdminRoles:
+      role1: "Company Admin"
```
Comment (ntruchsess) [inline charts/portal/values.yaml:411]: this should be 'role0', shouldn't it?

**Review by ntruchsess — CHANGES_REQUESTED** [review]

File: charts/portal/values.yaml (hunk 1 above)
Comment (Phil91) [reply charts/portal/values.yaml]: right - fixed it.

**Review by ntruchsess — APPROVED** [review]

File: charts/portal/templates/deployment-backend-appmarketplace.yaml [hunk 2]
```diff
@@ -160,6 +160,8 @@ spec:
           value: "{{ .Values.backend.appmarketplace.submitAppNotificationTypeIds.type0 }}"
         - name: "APPMARKETPLACE__TECHNICALUSERPROFILECLIENT"
           value: "{{ .Values.backend.appmarketplace.technicalUserProfileClient }}"
+        - name: "APPMARKETPLACE__COMPANYADMINROLES_Cl2-CX-Portal__0"
```
Comment (ntruchsess) [inline charts/portal/templates/deployment-backend-appmarketplace.yaml:163]: missing underscore after 'ROLES'

**Review by ntruchsess — CHANGES_REQUESTED** [review]

File: charts/portal/templates/deployment-backend-appmarketplace.yaml (hunk 2 above)
Comment (Phil91) [reply charts/portal/templates/deployment-backend-appmarketplace.yaml]: Fixed

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**